deploy/restartComponent.sh

```diff
@@ -3,16 +3,17 @@
 HOST=`hostname`
 DATENOW=`date +%s`
 
+# Get a few environment variables in, like $install and $manage
+source /data/admin/wmagent/env.sh
+
 # Figure whether it's a python2 or python3 agent
-if [ -d "/data/srv/wmagent/current/install/wmagent" ]; then
-  WMA_PATH="/data/srv/wmagent/current/install/wmagent"
-else
-  WMA_PATH="/data/srv/wmagent/current/install/wmagentpy3"
+if [ ! -d "$install" ]; then
+  install="/data/srv/wmagent/current/install/wmagentpy3"
 fi
 
 COMPONENTS="ErrorHandler JobSubmitter AgentStatusWatcher"
```

Comment:
Why do we have those hard coded inside the script? Wouldn't it be better to have those as  options at call time?

Reply:
That's a good idea. I will change it soon.